routes/recaptcha_solver.js

```diff
@@ -23,21 +23,22 @@ const DEFAULT_TIMEOUT = 1000;  // 1 second
  */
 
 async function action(page, request) {
-
     let recaptcha_data;
 
     if (request.body.solve_recaptcha) {
         recaptcha_data = await page.solveRecaptchas();
-    }
-    else {
+    } else {
         recaptcha_data = await page.findRecaptchas();
     }
 
     const waitOptions = request.body.waitOptions || { timeout: DEFAULT_TIMEOUT };
-    const close_page = request.query.closePage ||
-        (request.body.close_on_empty && recaptcha_data['captchas'].length === 0)
+    if (request.query.closePage ||
+        (request.body.close_on_empty && recaptcha_data['captchas'].length === 0)) {
+        await page.close();
```

Comment:
should be after `getContents`

Reply:
You are right!
It's quite strange that I did not get any bad response during testing.
I will look through my recaptcha spiders.